src/duckling/canvas/tools/entity-creator-tool.ts

```diff
@@ -34,4 +47,26 @@ export class EntityCreatorTool extends BaseTool {
     get icon() {
         return "pencil";
     }
+
+    private _buildSelectionBox(canvasZoom : number) : DisplayObject {
+        let graphics : Graphics = null;
```

Comment:
A lot of this code is duplicated in entity-move-tool.ts. You should extract it somehow.